crates/aptos-api-tester/src/tests/nft_transfer.rs

```diff
@@ -0,0 +1,578 @@
+// Copyright © Aptos Foundation
```

Comment:
Could we rename this file to make it clear that it is token v1? We'll likely want to add token v2 at some point later.